bin/merge_featurecounts.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env/python
+#!/usr/bin/python
```

Comment:
Yup - either `#!/usr/bin/python` or `#!/usr/bin/env python` (note the space).

The second one is better really - `#!/usr/bin/python` forces use of the system version of python. `#!/usr/bin/env python` will work with conda / other installations of python.